src/list-of-meals/constantOfListOfMeal.tsx

```diff
@@ -3,3 +3,6 @@ export const iconEaten =
   'https://png.pngitem.com/pimgs/s/509-5099390_check-green-check-list-icon-hd-png-download.png';
 
 export const minNumMealsPerDay = 3;
+
+export const timeOfLastMeal = '21 : 00';
+export const defaultLatestTime = '23 : 20';
```

Comment:
should be without spaces, since this is data layer, not UI layer

Reply:
I will keep this comment in mind for the future. In this project, such changes will lead to a lot of refinement.

Reply:
We agreed not to redo this as it brings lots of changes, but lesson learned: we should not rely on UI data to perform business logic.